cosmos/providers/databricks/notebook.py

```diff
@@ -9,9 +9,43 @@
 from airflow.providers.databricks.hooks.databricks import DatabricksHook
 from airflow.utils.context import Context
 from databricks_cli.runs.api import RunsApi
+from databricks_cli.sdk import JobsService
 from databricks_cli.sdk.api_client import ApiClient
 
 
+def _repair_task_retry_callback(context: Context) -> None:
+    """
+    This function allows the Airflow retry function to create a repair job for Databricks.
+    It uses the Databricks API to get the latest repair ID before sending the repair query.
+
+    Note that we use the `JobsService` class instead of the `RunsApi` class. This is because the
+    `RunsApi` class does not allow sending the `include_history` parameter which is necessary for
+    repair jobs.
+
+    Also for the moment we don't allow custom retry_callbacks. We might implement this in
+    the future if users ask for it, but for the moment we want to keep things simple while the API
+    stabilizes.
+    :return: None
+    """
+    current_task = context["ti"].task
+    api_client = current_task._get_api_client()
+    jobs_service = JobsService(api_client)
+    current_job = jobs_service.get_run(
+        run_id=current_task.databricks_run_id, include_history=True
+    )
```

Comment:
Would it make sense for us to check in advance if there is an ongoing repair and give a custom error message, or are we happy to return Databricks error stack traces?